config/config.go

```diff
@@ -26,6 +26,10 @@ type RedisConfig struct {
 	Password string
 }
 
+type HttpGetConfig struct {
+	Url string
```

Comment:
Convention: Use uppercase for abbreviations

```suggestion
	URL string
```